tests/conftest.py

```diff
@@ -1,4 +1,5 @@
-"""Ruff is forcing me to write a docstring for conftest.py."""
+"""Fixtures used in our test suite for the pyOpenSci Python package
+template."""
```

Comment:
When it starts working, pydoclint should complain about this line 😛. 

```suggestion
"""Provide fixtures to the entire test suite."""
template."""
```

Reply:
😆 sounds good!